Docs/source/Overview/getting-started.rst

```diff
@@ -4,8 +4,7 @@ Getting Started
 
 To work with the BPS, download the following software: 
 
-* `Kicad PCB Editor <https://kicad-pcb.org/>`_ - View :abbr:`SCH (pending)` and PCBs
+* `Kicad PCB Editor <https://kicad-pcb.org/>`_ - View :abbr:`SCH (Circuit Schematic File)` and PCBs
```

Comment:
Is this abbr definition needed or are even beginners expected to know what an sch file is?

Reply:
They don't need to know the abbreviation. I think "Circuit Schematic" should be sufficient.